Index: app/api/v1/mail.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/api/v1/mail.py b/app/api/v1/mail.py
new file mode 100644
--- /dev/null	(date 1764068648977)
+++ b/app/api/v1/mail.py	(date 1764068648977)
@@ -0,0 +1,13 @@
+import json
+from dotenv import load_dotenv
+from fastapi import APIRouter, WebSocket
+from langchain_openai import ChatOpenAI, OpenAIEmbeddings
+from langchain_core.prompts import ChatPromptTemplate, MessagesPlaceholder
+from langchain_core.messages import HumanMessage, AIMessage
+from langchain_core.output_parsers import JsonOutputParser
+from langchain_community.vectorstores import Chroma
+from langchain_community.document_loaders import TextLoader
+from langchain_text_splitters import RecursiveCharacterTextSplitter
+from langchain.chains import create_history_aware_retriever, create_retrieval_chain
+from langchain.chains.combine_documents import create_stuff_documents_chain
+from pydantic import BaseModel, Field
\ No newline at end of file
